src/components/home/Value.jsx

```diff
@@ -2,12 +2,12 @@ import React from "react";
 
 const Value = ({ title, description, icon }) => {
   return (
-    <div className="value-container group bg-gradient-to-t from-isa-blue-200 to-isa-blue-100 text-white text-3xl rounded-2xl h-72 w-72 shadow-black shadow-inner flex flex-col items-center mb-8 pt-4 hover:from-isa-yellow-200 hover:to-isa-yellow-100">
+    <div className="value-container group bg-gradient-to-t from-isa-blue-200 to-isa-blue-100 text-white text-lg md:text-3xl rounded-2xl h-80 w-80 shadow-black shadow-inner py-8 flex flex-col items-center hover:from-isa-yellow-200 hover:to-isa-yellow-100">
       {icon}
-      <div className="w-full text-center pt-4 text-white size-9 font-extrabold tracking-wider">
+      <div className="w-full text-center pt-3 text-white size-15 font-extrabold tracking-wider">
         {title}
       </div>
-      <div className="w-full text-center px-10 pt-8 text-white text-base font-semibold tracking-wider">
+      <div className="w-full text-lg text-center px-10 pt-8 text-white text-base font-semibold tracking-wider">
```

Comment:
<img width="731" alt="image" src="https://github.com/acm-ucr/isa-website/assets/91030482/77b5b25e-43d5-4aac-b546-096b586b9f50">

bring the description up a little bit more. might be because of the top padding.